tests/cases/helpers/case_helpers_skilled_nursing_facility.yaml

```diff
@@ -26,4 +26,98 @@ LoginSNFHelper:
     # Verify that the user has successfully logged in.
     - Type: wait_for
       Strategy: xpath
-      Id: $PAGE.snf_home.new_request_btn$
\ No newline at end of file
+      Id: $PAGE.snf_home.new_request_btn$
+
+SNFNewRequestHelper:
```

Comment:
Let's add a comment explaining the kind of request this is. Is is a provideer request?

Reply:
I think all SNF requests are to providers, but will add the comment